app/src/main/java/org/fossasia/phimpme/leafpic/util/AlertDialogsHelper.java

```diff
@@ -61,6 +64,35 @@ public static AlertDialog getInsertTextDialog(final ThemedActivity activity, Ale
         dialogBuilder.setView(dialogLayout);
         return dialogBuilder.create();
     }
+    public static AlertDialog getInsertTextDialogLink(final ThemedActivity activity, AlertDialog.Builder dialogBuilder , EditText editText, String link) {
```

Comment:
@vinaysajjanapu Yes, I am aware of that. Since I wanted to create an alert dialog box with a clickable link, it wouldn't be possible in the existing AlertDialogBox.